src/tests/test_sqlthread.py

```diff
@@ -5,40 +5,273 @@
 import threading
 import unittest
 
-from .common import skip_python3
-
-skip_python3()
 
 os.environ['BITMESSAGE_HOME'] = tempfile.gettempdir()
-
-from pybitmessage.helper_sql import (
-    sqlQuery, sql_ready, sqlStoredProcedure)  # noqa:E402
-from pybitmessage.class_sqlThread import sqlThread  # noqa:E402
+from pybitmessage.class_sqlThread import TestDB  # noqa:E402
 from pybitmessage.addresses import encodeAddress  # noqa:E402
 
 
+def filter_table_column(schema, column):
+    """
+        Filter column from schema
+    """
+    for x in schema:
+        for y in x:
+            if y == column:
+                yield y
+
+
 class TestSqlThread(unittest.TestCase):
```

Comment:
should be called something like `TestUpgradeDB`

Reply:
Done

Reply:
Done